dev-support/cicd/README.md

```diff
@@ -34,3 +41,22 @@ Enter Menu ID:1
 ==== Merge PR Begin ====
 Enter Your Apache JIRA User name:
 ```
+
+## Manual mode
+
+First, You need install `python 2.7.13` and `pip insalll jira`
```

Comment:
There's a  typo. The "installl" should be "install"

Reply:
@liuxunorg , could you please update the typo ?

Reply:
Done.